test/tests/functional/pbs_alps_request_tunables.py

```diff
@@ -0,0 +1,150 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+import math
+
+
+@tags('cray')
+class TestCrayAlpsRequestTunables(TestFunctional):
+    """
+    Set of tests to verify alps release tunables namely alps_release_interval
+    and alps_release_jitter
+    """
+
+    def setUp(self):
+        machine = self.du.get_platform()
+        if not machine.startswith('cray') or machine == "craysim":
+            self.skipTest("Test suite only meant to run on a Cray")
+        TestFunctional.setUp(self)
+
+    def get_epoch(self, msg):
+        a = time.strptime(msg[:19], "%m/%d/%Y %H:%M:%S")
```

Comment:
I'm not sure :19 will work.  Is the month, day, hour, and minute always 2 characters?  If not, you'll have a variable length time.  Since this is coming from a log message, you can always split on the ';'